src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -201,6 +199,41 @@ public boolean accept(Path path) {
     }
 
     /**
+     * Propagate any values that need to be passed to Hadoop-BAM through configuration properties:
+     *
+     *   - the validation stringency property is always set
+     *   - if the input file is a CRAM file, the reference value, which must be a URI which includes
+     *     a scheme, will also be set
```

Comment:
Also mention that the method will throw if the reference is null and the input is a CRAM